lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
It sounds like we never want to return `UNDEFINED`? Any timeout operation would result in a cancelled operation -- is that what we want?

* If indeed we never want to return `UNDEFINED`, we should just never return it in the first place (we would save on extra logic, DAO calls, etc.)?
* Otherwise, should we make it explicit that we want to cancel the operation (otherwise any GET will _fix_ it implicitly)?